### ISSUE TITLE
EDT позволяет одновременно установить свойства модуля "Внешнее соединение" и "Привилегированный"

### ISSUE THREAD

**Issue by VAGoncharov** [1C-Company/1c-edt-issues#845]
### Описание ошибки

Конфигуратор не позволяет проставить на общий модуль свойства "Внешнее соединение" и "Привилегированный", только что-то одно. EDT позволяет это сделать, вводя разработчика в заблуждение. При экспорте проекта EDT в cf конфигуратор сбрасывает одну из настроек. Это может привести к неявным для разработчика ошибкам.

### Как воспроизвести

1. Создать в EDT общий модуль.
2. Проставить в нем свойства "Внешнее соединение" и "Привилегированный".
3. Экспортировать конфигурацию в cf.
4. Убедиться, что конфигуратор сбросил галочку "Внешнее соединение".

### Ожидаемое поведение

EDT не позволяет установить на общий модуль одновременно "Внешнее соединение" и "Привилегированный"

### Лог рабочей области

-

### Версия 1С:EDT

Ruby 2022.1.5

### Операционная система

Windows

### Установленные плагины

1C:Code style V8, 1C:SSL-support

**Comment by adminimusRU** [comment]
EDT данную ошибку корректно диагностирует и выводит маркер на модуле:
![image](https://user-images.githubusercontent.com/28653631/215471283-7add0e0a-f7ab-4322-ba09-28035adb1969.png)
Сама проверка реализована в плагине code style еще с версии 0.1.

**Comment by NikitaMikhaylovSB** [comment]
Ну, проверка и разрешение так сделать - все-таки разные вещи)
EDT не должно давать такое установить.

**Comment by VAGoncharov** [comment]
Это должно быть реализовано на уровне EDT, а не в виде проверки плагина. При экспорте в cf галка все равно слетит, толку от валидатора?
К тому же, у нас у многих проверки v8-code-style отключены пока что, т.к. отнимают слишком много ресурсов локальной тачки, проект проверяется в сонаре.

**Comment by adminimusRU** [comment]
Согласен, проверка в плагине не полноценная замена. Допустимые сочетания галок описаны в конце раздела 5.5.2.1 документации платформы, можно однозначно продублировать поведение конфигуратора в EDT.
